ui/pages/Search/components/SavedSearch.jsx

```diff
@@ -78,8 +78,8 @@ const SavedSearches = ({ options, savedSearchesByGuid, selectedSearch, load, loa
       loading={loading}
       placeholder="Select a Saved Search"
       value={(selectedSearch || {}).savedSearchGuid}
-      onChange={val => onChange(savedSearchesByGuid[val].search)}
-      options={options}
+      onChange={val => (val ? onChange(savedSearchesByGuid[val].search) : onChange({}))}
+      options={[{ text: 'None', value: null, category: 'Default Searches' }].concat(options)}
```

Comment:
you should never construct new arrays in a prop definition, you should update the selector that creates `options` to include all the options you want it to have